contracts/contracts/strategies/BaseConvexMetaStrategy.sol

```diff
@@ -140,6 +140,39 @@ abstract contract BaseConvexMetaStrategy is BaseCurveStrategy {
         balance = balance.scaleBy(assetDecimals, 18) / THREEPOOL_ASSET_COUNT;
     }
 
+    /**
+     * @dev This function is completely analogous to _calcCurveTokenAmount[BaseCurveStrategy]
+     * and just utilizes different Curve (meta)pool API
+     */
+    function _calcCurveMetaTokenAmount(uint128 _coinIndex, uint256 _amount)
+        internal
+        returns (uint256 requiredMetapoolLP)
+    {
+        uint256[2] memory _amounts = [uint256(0), uint256(0)];
+        _amounts[uint256(_coinIndex)] = _amount;
+
+        // LP required when removing required asset ignoring fees
+        uint256 lpRequredNoFees = metapool.calc_token_amount(_amounts, false);
+        // LP required if fees would apply to entirety of removed amount
+        uint256 lpRequiredFullFees = lpRequredNoFees.mulTruncateScale(
+            1e10 + metapool.fee(),
```

Comment:
I couldn't find the fee method in the convex docs or contract code to verify it is using scale 1e10. Where is it?

Reply:
Here it is: https://curve.readthedocs.io/exchange-pools.html#StableSwap.fee

Reply:
Perhaps, can add this link as a comment, so that it'd be easier to refer in future

Reply:
sure adding that in